src/Actions/Actions/SelectAction.cs

```diff
@@ -67,6 +67,7 @@ public double IntervalMouseSelector
         // Backing object for POIElementContext - is disposed via POIElementContext property
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2213:DisposableFieldsShouldBeDisposed", MessageId = "_POIElementContext")]
         ElementContext _POIElementContext = null;
+        private readonly Object _elementContextLock = new object();
```

Comment:
>O [](start = 25, length = 1)

Does VS suggest object (with a lower case) here? One's an alias of the other, and I know that VS often suggests the use of the lower case version.

Reply:
I've made the change to lowercase 'o'. I thought about using "_lockObject," which I think is fine. But I didn't see the harm in being more specific.